Replicator/Pusher.cc

```diff
@@ -225,6 +225,10 @@ namespace litecore { namespace repl {
     // Sends a "changes" or "proposeChanges" message.
     void Pusher::sendChanges(RevToSendList &changes) {
         MessageBuilder req(_proposeChanges ? "proposeChanges"_sl : "changes"_sl);
+        if(_proposeChanges) {
+            req[kConflictIncludesRevProperty] = "true"_sl;
```

Comment:
The property name was based on the spec I wrote and send out in an email for feedback.  If it changes now it will need a Sync Gateway patch as well (for either of these changes).  Do you still want to go forward with that?

Reply:
No, oh well! I'm sorry I didn't catch this earlier.